apps/vite/src/components/calendar/UpcomingEvent.tsx

```
@@ -1,21 +1,19 @@
-import React from 'react'
-import { classNames } from '../../helper/utils'
+import React from "react"
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)